### DISCUSSION TITLE
Litvar

### DISCUSSION THREAD

**Discussion by jwehr618** [KarchinLab/open-cravat#242] [Usability & Feature Requests]
Hello,

Would you be able to add the pubmed ID of the papers pulled from litvar in addition to the rsID that is already available? 

Thanks

**Comment by kmoad** [comment]
We are working on adding this.

**Comment by kmoad** [comment]
We have added a new annotator `litvar_full` that includes the number of litvar references found and a list of all litvar pmids. Because of limitations with litvar's api, it only finds coding sequence variants. Also, it is live querying litvar's api using the dbsnp rsid to get the full pmid list. So please be aware that it has the potential to leak information outside of OpenCRAVAT.
